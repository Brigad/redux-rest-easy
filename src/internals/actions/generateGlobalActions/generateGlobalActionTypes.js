import generateActionType from '../../utils/generateActionType';

const generateGlobalActionTypes = () => ({
  RESET_ALL: generateActionType('@global', '@reset', 'RESET_ALL'),
});

export default generateGlobalActionTypes;
