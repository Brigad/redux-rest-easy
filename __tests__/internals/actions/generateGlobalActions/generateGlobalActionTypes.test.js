import generateGlobalActionTypes from '../../../../src/internals/actions/generateGlobalActions/generateGlobalActionTypes';

describe('generateGlobalActionTypes', () => {
  test('only path', () => {
    expect(generateGlobalActionTypes()).toMatchSnapshot();
  });
});
