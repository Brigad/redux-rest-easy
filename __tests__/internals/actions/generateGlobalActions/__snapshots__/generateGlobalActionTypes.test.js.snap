// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generateGlobalActionTypes only path 1`] = `
Object {
  "RESET_ALL": "@@rest-easy/@global/@reset/RESET_ALL",
}
`;
